packages/profile-sync-controller/src/controllers/user-storage/services.ts

```diff
@@ -240,6 +240,35 @@ export async function batchUpsertUserStorage(
   }
 }
 
+/**
+ * User Storage Service - Delete Storage Entry.
+ *
+ * @param opts - User Storage Options
+ */
+export async function deleteUserStorage(
+  opts: UserStorageOptions,
+): Promise<void> {
+  const { bearerToken, path, storageKey } = opts;
+  const encryptedPath = createEntryPath(path, storageKey);
+  const url = new URL(`${USER_STORAGE_ENDPOINT}/${encryptedPath}`);
+
+  const userStorageResponse = await fetch(url.toString(), {
+    method: 'DELETE',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${bearerToken}`,
+    },
+  });
+
+  if (userStorageResponse.status === 404) {
+    throw new Error('user-storage - feature/entry not found');
+  }
+
+  if (userStorageResponse.status !== 200 || !userStorageResponse.ok) {
+    throw new Error('user-storage - unable to delete data');
+  }
```

Comment:
NEED TO CHECK - sometimes our endpoints return a 204 No Content, I wonder if delete returns a 200 or a 204?